components/automate-ui/src/app/modules/desktop/insight/insight.component.ts

```diff
@@ -1,11 +1,12 @@
 import { Component, HostBinding, Input, Output, OnInit, EventEmitter } from '@angular/core';
 import { Store, createSelector } from '@ngrx/store';
 import { NgrxStateAtom } from 'app/ngrx.reducers';
-import { Desktop, TermFilter } from 'app/entities/desktop/desktop.model';
+import { Desktop, TermFilter, NodeMetadataCount, NodeMetadataCountValue } from 'app/entities/desktop/desktop.model';
 import { FilterUpdate } from 'app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.model';
 import { finalize } from 'rxjs/operators';
 import * as moment from 'moment/moment';
 import { saveAs } from 'file-saver';
+import { find } from 'lodash/fp';
```

Comment:
Wish there was a way to automatically order imports... Suggest organize all 3rd party ones in the first group (so move all these up a few lines), then chef ones in the second group.